variables.tf

```
@@ -330,6 +330,18 @@ variable "tags" {
   default     = {}
 }
 
+variable "terraformer_permissions_boundary_policy_description" {
+  type        = string
+  description = "The description to associate with the IAM permissions boundary policy attached to the Terraformer instance role in order to protect the foundational resources deployed in this account."
+  default     = "Defines the IAM permissions boundary for Terraformer instances in order to protect the foundational resources deployed in this account."
```

I think this language from the `description` field is clearer and can simply be repeated:
```suggestion
  default     = "The IAM permissions boundary policy attached to the Terraformer instance role in order to protect the foundational resources deployed in this account."
```